src/pages/excursions.js

```diff
@@ -1,87 +1,84 @@
 import React from 'react';
-import Cabin from '../images/cabin.jpg'; 
+import Cabin from '../images/cabin.jpg';
 import '../App.css';
 
+const Excursions = () => {
+	return (
+		<div className='Container'
+			style={{
+				backgroundColor: '#B4BF5E',
+				marginTop: '-21px',
+				marginLeft: '282px',
+				marginRight: '282px',
+				height: '180vh'
```

Comment:
Don't mix styles. Doing it this way is fine (although I suggest looking into "React styled components"), but don't use both this and seperate CSS files :)